src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/OverlappingTrainingDateRequestController.cs

```diff
@@ -47,13 +47,59 @@ IFeatureTogglesService<ProviderFeatureToggle> featureTogglesService
             _featureTogglesService = featureTogglesService;
         }
 
+        [HttpGet]
+        [Route("overlap-options-with-pending-request")]
+        public IActionResult DraftApprenticeshipOverlapOptionsWithPendingRequest(DraftApprenticeshipOverlapOptionWithPendingRequest request)
+        {
+            var featureToggleEnabled = _featureTogglesService.GetFeatureToggle(ProviderFeature.OverlappingTrainingDateWithoutPrefix).IsEnabled;
+            var vm = new DraftApprenticeshipOverlapOptionWithPendingRequestViewModel()
+            {
+                CohortReference = request.CohortReference,
+                DraftApprenticeshipHashedId = request.DraftApprenticeshipHashedId,
+                DraftApprenticeshipId = request.DraftApprenticeshipId,
+                OverlappingTrainingDateRequestToggleEnabled = featureToggleEnabled,
```

Comment:
this property isn't used in the view so why are we setting it in the viewmodel?

Reply:
Updated